addons/website/controllers/main.py

```diff
@@ -238,10 +238,10 @@ def create_sitemap(url, content):
 
     @http.route('/website/info', type='http', auth="public", website=True, sitemap=True)
     def website_info(self, **kwargs):
-        try:
-            request.website.get_template('website.website_info').name
-        except Exception as e:
-            return request.env['ir.http']._handle_exception(e)
+        if not request.website.viewref('website.website_info', False).active:
```

Comment:
Maybe mention the opw in this commit message too?

Reply:
Sure, should be here too